changelog/_unreleased/2021-02-18-add-retryable-transaction-to-prevent-deadlocks.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Adds the RetryableTransaction to prevent database deadlocks
+issue:
```

Comment:
@hanneswernery, you are [missing the author and author_email field](https://github.com/shopware/platform/blob/trunk/adr/2020-08-03-Implement-New-Changelog.md). 
If you want you can just send them to me and i add them together with the issue-key. (you can just user your github email, sorry that i didn't notice that earlier, i thought these fields were there : /)

Reply:
You are right. I just added the necessary meta data to the changelog.
You only have to add the respective issue key 🙂